main.css

```diff
@@ -60,6 +60,13 @@ a:hover {
   text-transform: uppercase;
 }
 
+@media screen and (min-width: 60em) {
```

Comment:
This change doesn't make a lot of sense to me. I think that keeping `unbreakable` unbreakable *all the time* is a little easier to understand and is more predictable. I'm curious what specifically warranted this change and maybe we can come up with a better way to handle it.